package.json

```diff
@@ -3,18 +3,21 @@
   "version": "18.0.0",
   "description": "LazyLoad is a lightweight (2.4 kB) and flexible script that speeds up your web application by deferring the loading of your below-the-fold images, videos and iframes to when they will enter the viewport. It's written in plain \"vanilla\" JavaScript, it leverages the IntersectionObserver API, it supports responsive images, it optimizes your website for slower connections, and can enable native lazy loading.",
   "main": "dist/lazyload.min.js",
-  "module": "dist/lazyload.esm.js",
+  "module": "dist/esm/lazyload.js",
   "browser": "dist/lazyload.min.js",
   "typings": "typings/lazyload.d.ts",
+  "type": "module",
+  "sideEffects": false,
   "devDependencies": {
-    "@babel/core": "^7.13.14",
-    "@babel/plugin-transform-object-assign": "^7.12.13",
-    "@babel/preset-env": "^7.13.12",
-    "@rollup/plugin-babel": "^5.3.0",
-    "@rollup/plugin-node-resolve": "^11.2.1",
-    "jest": "^27.2.1",
-    "rollup": "^2.44.0",
-    "rollup-plugin-terser": "^7.0.2"
+    "@babel/core": "^7.24.3",
+    "@babel/plugin-transform-object-assign": "^7.24.1",
```

Comment:
Unrelated, but this is probably not needed anymore (nor in babelrc)

Reply:
I've kept the that module since it appears to be utilized here: https://github.com/verlok/vanilla-lazyload/blob/4e44c46eff6249c7ef4eb635139ebc89c11f5b64/rollup.config.js#L50-L53

It pertained to the "non" ESM module. I'm uncertain why it was present, whether it was for the utilization of the "plugin-transform-object-assign" plugin or for transpiling to a JavaScript language target version. Thus, I chose to retain it, but yes probably isn't required anymore.